gbpservice/nfp/service_plugins/firewall/nfp_fwaas_plugin.py

```diff
@@ -115,9 +116,11 @@ def nexthop_nets_query(nets, visited):
     nets = set([from_net_id])
     while nets:
         if to_net_id in nets:
+            context = original_context
```

Comment:
Yes, because nexthop_nets_query() needs to use admin context, which gets  called in loop. Approach we followed is use admin context in function, and wherever function is returning restore original context.